frontend/src/core/runtime/runtime.ts

```diff
@@ -104,6 +104,12 @@ export class RuntimeManager {
    * The URL of the copilot server.
    */
   getLSPURL(lsp: "pylsp" | "copilot"): URL {
+    if (lsp === "copilot") {
```

Comment:
Connection logic is here: https://github.com/marimo-team/marimo/blob/6405da156e5ec39d454db38bdbc0b11e741e86b3/lsp/index.ts#L143-L154